api_tests/test_endpoints.py

```diff
@@ -0,0 +1,37 @@
+import os
+import requests
+import json
+
+
+def run_this():
+    target_directory = os.path.join(os.getcwd(), "../json_files")
```

Comment:
I'm pretty sure this still won't work on windows. Literally CANNOT use "/" in all paths without using special libraries. 
To get to previous directory have to use os functions to do so.  I believe what you want is `os.path.join(os.getcwd(), os.getcwd(os.pardir),"json_files"`